content/en/blog/2020/new-deployment-model/index.md

```diff
@@ -0,0 +1,67 @@
+publishdate: 2020-08-25
+attribution: "Lin Sun (IBM), Iris Ding (IBM)"
+keywords: [istiod,deployment model,install,deploy,'1.7']
+---
+
+## Background
+
+From experience working with various service mesh users and vendors, we believe there are 3 key personas for a typical service mesh:
+
+* Mesh Operator, who manages the service mesh control plane installation and upgrade.
+
+* Mesh Admin, often referred as Platform Owner, who owns the service mesh platform and defines the overall strategy and implementation for service owners to adopt service mesh.
+
+* Mesh User, often referred as Service Owner, who owns one or more services in the mesh.
+
+It is common to have all these personas work on the same clusters without any clear separation.  For example, there are multiple ways to deploy Istio according to the [docs](/docs/setup/install/), which all start with mesh operator, mesh admin/user sharing the single cluster first and then gradually expanding the mesh to multiple clusters or VMs.  None of these provided a clear separation between mesh operator and mesh admin/user at the boundary of a cluster.  You may be thinking you could set up [Kubernetes RBAC rules](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) and namespaces to control which personas can do what within the cluster, however, sometimes you need stronger separation between the mesh control plane and the rest at the cluster level.
+
+We're introducing a new deployment model for Istiod which enables mesh operators to install and manage the mesh control plane on separate clusters. This new deployment model allows a clear separation between mesh operators and mesh admins. With this new model, Istio mesh operators can run Istio control plane for mesh admins while mesh admins can still control the configuration of the control plane without worrying about installing or managing the Istio control plane. This model is transparent to mesh users.
+
+## Control Plane Cluster Deployment model
+
+After installing Istio using the [default installation profile](/docs/setup/install/istioctl/#install-istio-using-the-default-profile), you have Istiod control plane installed in a single cluster like the diagram below:
+
+{{< image width="100%"
+    link="single-cluster.svg"
+    alt="Istio mesh in a single cluster"
+    title="Single cluster Istio mesh"
+    caption="Istio mesh in a single cluster"
+    >}}
+
+With this new deployment model in Istio 1.7, it is possible to run Istiod on a separate cluster (the `control plane cluster`) from services, as shown in the diagram below. The `control plane cluster` is owned by mesh operator while the mesh admin owns the cluster that has services deployed in the mesh. Mesh admin has no access to the `control plane cluster`. If you are interested in exploring this, you can follow the [central istiod single cluster step by step guide](https://github.com/istio/istio/wiki/Central-Istiod-single-cluster-steps).
```

Comment:
Reword to say that we can run istiod as an external control plane, for example running it in a cluster owned by the mesh operator. We can drop any mention of control plane cluster or primary cluster or whatever. Just say that the mesh operator owns the control plane, and the mesh admin has no access to it.

Then you can link to an example of how to run istiod as an external control plane for mesh operators to view.

Reply:
Thanks Sven!  Updated PTAL